include/envoy/event/file_event.h

```diff
@@ -18,15 +18,24 @@ struct FileReadyType {
   static const uint32_t Closed = 0x4;
 };
 
-enum class FileTriggerType { Level, Edge };
+#define FORCE_LEVEL_EVENTS
```

Comment:
Please remember to remove this define before declaring this PR done.  We should not switch to level events on linux and macos by default.  Keep comment open until the previous line is removed.